website/docs/components/flyout/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,26 @@
+## Conformance
+
+**Note:** Putting this in for now, can adjust when the component is built.
+
+<Doc::Badge @type='success'>Conformant</Doc::Badge>
+
+The Flyout component is conformant when used as directed.
+
+## Focus
+
+- When a Flyout is triggered, focus is trapped within the component.
+    - If there are no interactive elements within the body of the Flyout, the dismiss button should receive focus as the first interactive element in the [DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction).
+    - If there is an interactive element within the body (link a link), that element should receive focus first.
```

Comment:
Is this placeholder text? "(link a link)" If so it may be helpful to make it stand out more by adding "TODO:". You could also add an HTML comment with a TODO: instead so that it doesn't display within the web page content.

Reply:
Lol, not placeholder, just a typo. Should be "like a link", which is a common interactive element within the flyout body.

Reply:
I thought you were reminding yourself to add a link... [😂](https://emojipedia.org/face-with-tears-of-joy/)